### ISSUE TITLE
Add visuals to the book

### ISSUE THREAD

**Issue by leudz** [leudz/shipyard#55]

**Comment by dakom** [comment]
For inspiration, the "grokking [stuff]" series has a great visual teaching style, e.g. https://www.manning.com/books/grokking-algorithms

**Comment by griffi-gh** [comment]
yes! it's already pretty amazing, visuals will make it even better!
